swiper.el

```diff
@@ -503,6 +503,17 @@ When non-nil, INITIAL-INPUT is the initial search pattern."
   (interactive)
   (swiper--ivy (swiper--candidates) initial-input))
 
+;;;###autoload
+(defun swiper-region-or-not ()
```

Comment:
Why not just call it `swiper-region`? Or add this logic to `swiper` itself, based on a new user option?

Reply:
Fix to swiper-region and swiper-isearch-region.